back/src/config/pg/migrations/1715689358009_add-index-on-id-in-discussion-table.ts

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { MigrationBuilder } from "node-pg-migrate";
+
+const discussionsTableName = "discussions";
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.createIndex(discussionsTableName, "id");
```

Comment:
il s'agit d'une colonne "unique". Postgres crée déjà un index dessus:

> PostgreSQL automatically creates a unique index when a unique constraint or primary key is defined for a table
https://www.postgresql.org/docs/13/indexes-unique.html

Reply:
@bbohec @JeromeBu pour info, par rapport à notre discussion sur les index:
- avant ce script de migration on a un index de type UNIQUE sur la colonne id
- après ce script, on a un nouvel index ordinaire qui se crée sur la colonne id 

Petite différence sur la performance car UNIQUE vérifie aussi la contrainte d'unicité (donc c'est forcément plus long sur une requête d'update), ce que l'index régulier ne fait pas:

> Performance Impact:
Data Retrieval: Similarly improves query performance by enabling faster searches.
Data Modification: Additional overhead during inserts and updates due to the need to check for and enforce uniqueness, which can slightly slow down these operations compared to a regular index.
(merci chatgpt)